radix-engine/src/engine/execution_trace_module.rs

```diff
@@ -0,0 +1,143 @@
+use sbor::rust::collections::*;
+use scrypto::buffer::scrypto_decode;
+use scrypto::core::{FnIdentifier, NativeFnIdentifier, Receiver, VaultFnIdentifier};
+use scrypto::engine::types::*;
+use scrypto::prelude::{VaultPutInput, VaultTakeInput};
+use scrypto::values::*;
+
+use crate::engine::*;
+use crate::model::*;
+
+pub struct ExecutionTraceModule;
+
+impl ExecutionTraceModule {
```

Comment:
Seems that this should just be `ExecutionTrace`

Reply:
yeah, makes sense